### PR TITLE
Controls Buttons reposition / Multi-Selection Operation

### PR DESCRIPTION
**Pull request by Fia3D**
image

### CODE REVIEW

**Review by Fayti1703 — CHANGES_REQUESTED** [review]
Hate to be so immediately negative to a first-time contributor, but could you:

1. Correct the indentation on new lines
2. Remove the spurious changes on the lines just before the new lines (I'm guessing that's a CRLF/LF difference)
4. Split your changes across multiple commits (One atomic change per commit)

**Comment by Fia3D** [comment]
image
Last commit allow to remove the white circle (ctrl + click on rotation)

File: Ktisis/Util/EnumFlagExtension.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Ktisis.Util
+{
+	static class EnumFlagExtension
+	{
+		public static bool FlagHas<T>(this System.Enum type, T value) => ((int)(object)type & (int)(object)value) == (int)(object)value;
```
Comment (Fayti1703) [inline Ktisis/Util/EnumFlagExtension.cs:5]: This already exists as an instance method on `System.Enum` -- namely `HasFlag`. The other methods here are fine, but this one in specific already has a native implementation.

If you could change your calls to that method and then remove this method here, that'd be great.

**Review by Fayti1703 — CHANGES_REQUESTED** [review]
This is looking quite good! I would request one last change before calling this merge-ready:

**Comment by Fia3D** [comment]
Should be good now

**Review by Fayti1703 — APPROVED** [review]

**Comment by Fia3D** [comment]
Rebase to make it compatible with new merged titlebar settings

**Comment by Fayti1703** [comment]
Minor thing I just noticed: Could you un-hang the braces in your code? i.e.

```diff
- if (ImGui.GetIO().KeyCtrl)
- {
+ if (ImGui.GetIO().KeyCtrl) {
```

**Pull request merged**